docs/architecture/overview.md

```diff
@@ -0,0 +1,41 @@
+# Architecture Overview
+
+The Carbon Aware SDK was built in 3 layers, with each layer having it's own set of responsibilities. 
+1. User interface layer
```

Comment:
apologies if i'm opening a can of worms here, but any reason we can't call it the "consumer layer"?

Reply:
resolved by team vote